src/services/formatting.js

```diff
@@ -16,3 +17,21 @@ export function toFormattedText(text) {
     </span>
   ));
 }
+
+/*
+ * return <Tag>toFormattedText(Hello world)</Tag>
+ * for <tag>Hello world<tag>
+ */
+export function toFormattedTaggedText(text) {
+  const splitedText = text.split('<tag>').filter(line => line !== '');
+  const inTextBeginning = text.indexOf('<tag>') ? 1 : 0;
```

Comment:
This feels to me like we're re-doing some of React's built in rendering. Can't we just render the string as a React node/array of nodes with `<Tag>` nodes built-in instead of having to go write custom string parsing code?

Reply:
I am trying to know how to do that in a JSON file ... 😕
Do you know if that is possible?

Reply:
Hrmmm, that's a good question. I didn't see that you were trying to do this on pure JSON.

I did a little bit of googling, it looks like the [react-jsx-parser library](https://www.npmjs.com/package/react-jsx-parser) might do that though...

If it's too much trouble, I'd recommend skipping it though.

Reply:
Awesome it was easier to use this than parsing ...
I get rid of `formatting.js` in the last updated branch ... 
Do you want me to remove it in this one too or just wait for the last updated branch' PR

Reply:
@bencpeters ping 😁

Reply:
Let's remove it in this PR too.